jumpscale/core/identity/__init__.py

```diff
@@ -87,6 +98,13 @@ def explorer(self):
             from jumpscale.clients.explorer import export_module_as  # Import here to avoid circular imports
 
             ex_factory = export_module_as()
+
+            # Backward compitablity (Update mainnet url)
+            # Create new config has_migrated = True after mainnet update
+            if js_config.get("has_migrated", False):
```

Comment:
Can we rename it to a more specific name, to be related to this explorer URL change.
Also, we can add a note about better network selection implementation.